crypto/blake2/blake2b.c

```diff
@@ -14,10 +14,10 @@
  * can be found at https://blake2.net.
  */
 
+#include "e_os.h"
```

Comment:
Even though it might turn to be another diversion, do blake2[bs].c really need e_os.h? I seem to have wage memory that in previous version it was removing and me thinking "oh! that's good, algorithm implementations are last ones that would be OS-specific..."